exampleCourse/questions/demo/annotated/engCircuit/question.html

```diff
@@ -28,4 +28,4 @@
       </small>
       <pl-github-link></pl-github-link>
   </div>
-</pl-question-panel>
+</pl-question-panel>
```

Comment:
Let's retain the triling newline.

Reply:
@Anmol-Ghadia you should set your editor to add a trailing newline every time you save a file (and trim newlines beyond the first one). This is a pretty common convention across projects.